### ISSUE TITLE
Add Clojure 1.10

### ISSUE THREAD

**Issue by stuartstein777** [codewars/runner#6]
- Name: Clojure
- Version: 1.10

1.8 is getting a bit old now!

**Comment by hlegius** [comment]
It would be awesome, really! 👍

**Comment by bbatsov** [comment]
Now Clojure 1.11 is out as well https://clojure.org/news/2022/03/22/clojure-1-11-0

**Comment by port19x** [comment]
`1.11` added the `clojure.math` namespace, including a `round` function.
This would greatly reduce the work needed to complete many kata.

I'm sure a couple other improvements have made their way into the language as well, but thats the main thing I'm missing

**Comment by falcowinkler** [comment]
clojure 1.8 is getting _very_ old by now, it must be one of the oldest on codewars. The experience really is pretty bad if you need to look up the old way of doing things often. Any chance this can get some love @kazk?

**Comment by kazk** [comment]
Yeah, I've been wanting to update Clojure. The best way to contribute is to create a repository like https://github.com/codewars/riscv with an image and how to test. You can create one under your account first.

See https://github.com/codewars/runner/issues/241#issuecomment-1483420309 for the current version, which is not ideal. I'd like the new version to have a proper project setup with dependency management.

Please let me know if you have any questions or need help with anything.

**Comment by falcowinkler** [comment]
@kazk i created a clojure runner similar to the one you posted: https://github.com/falcowinkler/clojure-runner
Please let me know if you need support or any improvements on that, would love to see latest clojure on codewars ;)

**Comment by kazk** [comment]
@falcowinkler Thanks! I'll try to take a look sometime soon. Do you think you can set up GitHub Actions to run the examples so we can see the output on GitHub? You can copy https://github.com/codewars/riscv/blob/main/.github/workflows/ci.yml and adjust as needed.

**Comment by falcowinkler** [comment]
done @kazk : https://github.com/falcowinkler/clojure-runner/actions/runs/10201113439/job/28222201009

**Comment by falcowinkler** [comment]
@kazk let me know if I can support in any other way
